include/seqan3/range/views/take.hpp

```diff
@@ -314,16 +314,16 @@ class view_take<urng_t, exactly, or_throw>::iterator_type :
     }
 
     //!\brief Returns an iterator incremented by one.
-    constexpr iterator_type operator++(int) noexcept(noexcept(++std::declval<iterator_type &>()) &&
-                                           std::is_nothrow_copy_constructible_v<iterator_type>)
+    constexpr basic_iterator operator++(int) noexcept(noexcept(++std::declval<basic_iterator &>()) &&
+                                             std::is_nothrow_copy_constructible_v<basic_iterator>)
```

Comment:
I think this should be aligned to the second noexcept. 🧐
```suggestion
    constexpr basic_iterator operator++(int) noexcept(noexcept(++std::declval<basic_iterator &>()) &&
                                                      std::is_nothrow_copy_constructible_v<basic_iterator>)
```